extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/counters/CounterTracker.java

```diff
@@ -112,6 +112,16 @@ public <T extends QueryCounter> T counter(final String counterName, final Suppli
     return (T) countersMap.computeIfAbsent(counterName, ignored -> newCounterFn.get());
   }
 
+  public long totalCpu()
+  {
+    long cpuTimeNs = 0;
+    CpuCounters cpuCounters = (CpuCounters) countersMap.getOrDefault(CounterNames.cpu(), new CpuCounters());
```

Comment:
nit: it feels odd to use `getOrDefault` here...
if it would return the default - don't think that would have any `counters` in it - as nobody would call `forName` to create them